component/service/src/main/java/org/exoplatform/social/rest/suggest/PeopleRestServices.java

```diff
@@ -255,8 +256,8 @@ public Response getSuggestions(@Context SecurityContext sc, @Context UriInfo uri
       }
       for (Entry<Identity, Integer> suggestion : suggestions.entrySet()) {
         Identity id = suggestion.getKey();
-
-        if (id.getRemoteId().equals(userACL.getSuperUser())) continue;
+        boolean isExternal = id.getProfile().getProperty(Profile.EXTERNAL) != null && (id.getProfile().getProperty(Profile.EXTERNAL)).equals("true");
+        if ((id.getRemoteId().equals(userACL.getSuperUser())) || isExternal) continue;
         JSONObject json = new JSONObject();
         Profile socialProfile = id.getProfile();
```

Comment:
move this instruction above and use directly socialProfile for checking the isExternal property.

Reply:
Done !